modules/serverless-ossm-setup.adoc

```diff
@@ -1,12 +1,21 @@
-// Module included in the following assemblies:
-//
-// * serverless/networking/serverless-ossm.adoc
-
 [id="serverless-ossm-setup_{context}"]
-= Using {ProductShortName} with {ServerlessProductName}
+= Integrating {ProductShortName} with {ServerlessProductName}
+
+You can integrate {ProductShortName} with {ServerlessProductName} without using Kourier by completing the following procedure.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} on your {product-title} cluster.
+* You have installed {ProductName}. {ServerlessProductName} with {ProductShortName} only is supported for use with {ProductName} version 2.0.5 or newer.
```

Comment:
s/newer/higher ?

Reply:
Should _Creating a certificate to encrypt incoming external traffic_ be a prereq?

Reply:
> Should Creating a certificate to encrypt incoming external traffic be a prereq?

Good question - thoughts @cardil @markusthoemmes ?